code/game/gamemodes/changeling/changeling.dm

```diff
@@ -312,7 +312,7 @@ var/list/possible_changeling_IDs = list("Alpha","Beta","Gamma","Delta","Epsilon"
 		to_chat(user, "<span class='warning'>[T] is not compatible with our biology.</span>")
 		return
 
-	if((NOCLONE || SKELETON || HUSK) in T.mutations)
+	if((target.has_trait(TRAIT_SKELETON)) || (target.has_trait(TRAIT_NOCLONE)) || (target.has_trait(TRAIT_HUSK)))

```

Comment:
I might consider adding in an ORTRAIT and/or and ANDTrait variant to simplify this sort of check in the future

Reply:
Probably a separate function so I can pass in a list